packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -273,14 +273,13 @@ module.exports = {
       throw new ApplicationError('Register action is currently disabled');
     }
 
+    const { register } = strapi.config.get('plugin.users-permissions');
+
     const params = {
-      ..._.omit(ctx.request.body, [
-        'confirmed',
-        'blocked',
-        'confirmationToken',
-        'resetPasswordToken',
-        'provider',
-      ]),
+      ..._.pick(
+        ctx.request.body,
+        register?.allowedFields ? register?.allowedFields : ['username', 'email', 'password']
```

Comment:
What's this `register?.allowedFields`?

Reply:
New plugin configuration option, basically checking if the user has modified it and if not, using the defaults.

Instead of forcing the user to only accept what we allow, it grants them the freedom to configure it themselves.

Reply:
Should 'picture' be allowed by default?

Reply:
> Should 'picture' be allowed by default?

Picture isn't a default field on user, is it?

Reply:
You're right, I was looking at getstarted 🤦